salmon/test.py

```diff
@@ -216,7 +216,9 @@ def test_interpret(self):
                 self.assertTrue(isinstance(v, Quantitative))
         
 iris = pd.read_csv("https://raw.githubusercontent.com/uiuc-cse/data-fa14/gh-pages/data/iris.csv")
-
+commprop = pd.read_csv("https://raw.githubusercontent.com/uiuc-cse/data-fa14/gh-pages/data/CommProp.csv")
+plastic = pd.read_csv("https://raw.githubusercontent.com/uiuc-cse/data-fa14/gh-pages/data/Plastic.csv")
+realestate = pd.read_csv("https://raw.githubusercontent.com/uiuc-cse/data-fa14/gh-pages/data/Real Estate5.csv")
```

Comment:
As mentioned in our emails, please change these from url references to point to the data/ directory. If you could download the iris dataset and add it to the data/ directory and make the change to line 218 that would be appreciated as well.